core/crypto/abac/proto/abac.proto

```diff
@@ -0,0 +1,35 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
should use https://github.com/hyperledger/fabric/blob/master/docs/dev-setup/headers.txt replacing "[COPYRIGHT OWNER]" with "IBM Corp. 2016"